### ISSUE TITLE
Add 'none' value to AccessibilityFeature options

### ISSUE THREAD

**Issue by carakey** [osulp/Scholars-Archive#2761]
### Descriptive summary

Add an option for the `AccessibilityFeature` property CV for "none".

This is an authorized value in the source schema.org vocabulary (4.2.8), which we did not include in the original request. Since then, we have decided to use "none" for works that will be marked as archived and otherwise not evaluated for accessibility compliance. 

### Expected behavior

The drop-down for Accessibility Feature includes a value option of "none".

### Related work

#2710 

### Accessibility Concerns

none

**Comment by lamtu1** [comment]
@carakey I just updated the image to `staging` it should be up now. Can you give it another go when you get a chance? Thank you!

**Comment by carakey** [comment]
I'm unable to fully QA because I cannot save new or edited works on Staging (#2774)

**Comment by carakey** [comment]
QA pass
